build.template

```diff
@@ -1,7 +1,60 @@
-//! Configuration settings for this Rhai build
-//!
-//! This file is auto-generated from `build.template`
+#[cfg(feature = "stable_hash")]
+use core::cell::OnceCell;
 
+// [236,800,954,213], haha funny yume nikki reference
+
+#[cfg(not(feature = "stable_hash"))]
+/// Fixed hashing seeds for stable hashing.
+///
+/// Set to [`None`] to disable stable hashing.
+///
+/// See [`set_rhai_ahash_seed`] for more.
+///
+/// You can also set this at compile time by disabling `stable_hash` and setting the `RHAI_AHASH_SEED`
+/// environment variable instead.
+///
+/// E.g. `env RHAI_AHASH_SEED ="[236,800,954,213]"`
+const AHASH_SEED: Option<[u64; 4]> = {{AHASH_SEED}};
+#[cfg(feature = "stable_hash")]
 /// Fixed hashing seeds for stable hashing.
+///
 /// Set to [`None`] to disable stable hashing.
-pub const AHASH_SEED: Option<[u64; 4]> = {{AHASH_SEED}};
+///
+/// See [`set_rhai_ahash_seed`] for more.
+///
+/// You can also set this at compile time by disabling `stable_hash` and setting the `RHAI_AHASH_SEED`
+/// environment variable instead.
+///
+/// E.g. `env RHAI_AHASH_SEED ="[236,800,954,213]"`
+static AHASH_SEED: OnceCell<Option<[u64; 4]>> = OnceCell::new();
```

Comment:
I wonder if there is a way to fallback to a default seed if the user does not call `set_ahash_seed`?  I guess it might not be very useful though because the compiler really needs to get this default from somewhere.

Reply:
Ill just add a default to `get_ahash`.